src/oumi/cli/fetch.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2025 - Oumi
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Annotated, Optional
+
+import requests
+import typer
+import yaml
+
+from oumi.cli.cli_utils import resolve_oumi_prefix
+from oumi.utils.logging import logger
+
+OUMI_GITHUB_RAW = "https://raw.githubusercontent.com/oumi-ai/oumi/main/configs/recipes"
```

Comment:
```suggestion
OUMI_GITHUB_RAW = "https://raw.githubusercontent.com/oumi-ai/oumi/main"
```

Let's make this the top level directory so users can request samples from `/configs/examples` or `/configs/projects` as well